application/src/main/scala/com/azavea/franklin/database/StacCollectionDao.scala

```diff
@@ -21,24 +24,76 @@ object StacCollectionDao extends Dao[StacCollection] {
     selectF.query[StacCollection].to[List]
   }
 
+  def updateExtent(
+      collectionId: String,
+      bulkExtent: BulkExtent
+  ): ConnectionIO[Int] =
+    (OptionT(getCollection(collectionId)) flatMap { collection =>
+      val existingExtent = collection.extent
```

Comment:
Hm, should not it be just some:

```scala
collection.extent.expandToInclude(bulkExtent)
```

I mean to move the bbox expand logic out of the `Dao`

Reply:
Can the `BulkExtent` be encoded as the `Stac Extent`?

Reply:
I think so! I'll have to require a nonemptylist to make sure I can produce one, but that's no problem. That's a good change.

Reply:
Wait... what does that get me? This exists wholly for the db update to make a new collection. I'll just have to convert it to a bbox at the end anyway. The non-emptiness is still useful though.

Reply:
@jisantuc tbh my comment was in general more about [these lines](https://github.com/azavea/franklin/blob/17e428aa288add2383473d1aaf269a56e0a82cea/application/src/main/scala/com/azavea/franklin/database/StacCollectionDao.scala#L33-L69) that can live outside of the DAO.

Reply:
Oh, well they _can_ live out of the Dao, but that's just indirection righ tnow, since that's the only place it's needed. If it ends up being useful somewhere else then I think it makes sense to move it.

Reply:
I converted to a NEL in 688c9e4 and it at least gets rid of the option handling in there

Reply:
nice!